app/lib/vendor_api/changes/notes_for_applications.rb

```diff
@@ -0,0 +1,11 @@
+module VendorAPI
+  module Changes
+    class NotesForApplications < VersionChange
```

Comment:
do we need this change module, I think the single application will allow this to appear everywhere 🤔 ?

Reply:
I think both this and `app/lib/vendor_api/changes/notes_for_application.rb` are redundant. 
`Notes` are introduced as a nested resource in applications when a note is created (as the response JSON should contain an application with the nested comments), so the resource configuration can be moved in `app/lib/vendor_api/changes/create_note.rb`